src/object_storage/do_space.rs

```diff
@@ -44,7 +44,7 @@ pub(crate) async fn download_space_object(
                     Ok(_) => info!("File {} is well downloaded", path_to_download),
                     Err(e) => error!("{:?}", e),
                 },
-                Err(e) => error!("Unable to create file"),
+                Err(e) => error!("Unable to create file, maybe this file already exists..."),
```

Comment:
can you please be more explicit on the context. like adding file name, why etc..

Reply:
this fn is refactored,  now we only download it if file does't exist